dotnet/ios/ios12/MarsHabitatPricePredictor/MarsHabitatPricePredictor/DataSources/GreenhousesDataSource.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace MarsHabitatPricePredictor.DataSources;
+
+/// <summary>
+/// Data source for the number of greenhouses.
+/// </summary>
+public class GreenhousesDataSource
+{
+    /// <summary>
+    /// Possible values for greenhouses in the habitat
+    /// </summary>
+    public double[] Values { get; } = { 1, 2, 3, 4, 5 };
+
+    public string? Title (int index) =>
```

Comment:
Our framework naming conventions says we should use verbs for methods:

```suggestion
    public string? GetTitle (int index) =>
```

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods